src/linux/process.rs

```diff
@@ -219,6 +221,20 @@ impl ProcessExt for Process {
     fn group_id(&self) -> Option<Gid> {
         self.group_id
     }
+
+    fn wait(&self) -> Option<Pid> {
+        let (mut status, mut _pid): (libc::c_int, i32) = (0, -1);
+        let _ppid = self.parent().unwrap().as_u32();
```

Comment:
I'm really having cold feet about this because it's because that a process gets detached from its parent. I think the best solution is to just loop on the `kill` call and wait between each try.